views/article.html

```diff
@@ -80,6 +81,15 @@
 			}
 			</script>
 		</amp-analytics>
+		<amp-analytics type="krux" config="https://cdn.krxd.net/controltag/amp/KHUSeE3x.json">
+			<script type="application/json">
+			{
+				"extraUrlParams": {
+						"page.topics": "UK welfare reform,Minimum-Living Wage"
+					}
```

Comment:
https://github.com/Financial-Times/n-ui/blob/master/ads/js/oAdsConfig.js#L73

Is it meant to be smth along these lines? guess if reinstate backend call to context can actually add a lot of data? Not going to work with user call though? Or can we access the cookie?